lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -136,19 +136,23 @@ class FlGridData {
 
 /// This class can be used wherever we want draw a straight line,
 /// and contains visual properties
+/// [dashArray]  A circular array of dash offsets and lengths.
+/// For example, the array `[5, 10]` would result in dashes 5 pixels long
+/// followed by blank spaces 10 pixels long.  The array `[5, 10, 5]` would
+/// result in a 5 pixel dash, a 10 pixel gap, a 5 pixel dash, a 5 pixel gap,
+/// a 10 pixel dash, etc.
 class FlLine {
   final Color color;
   final double strokeWidth;
+  final List<int> dashArray;
 
-  const FlLine({
-    this.color = Colors.black,
-    this.strokeWidth = 2,
-  });
+  const FlLine({this.color = Colors.black, this.strokeWidth = 2, this.dashArray});
 
   static FlLine lerp(FlLine a, FlLine b, double t) {
     return FlLine(
       color: Color.lerp(a.color, b.color, t),
       strokeWidth: lerpDouble(a.strokeWidth, b.strokeWidth, t),
+      dashArray: b.dashArray,
```

Comment:
Please use `lerpDoubleList` (in the `lerp.dart`),
It has a better experience when we animate the chart, 
The dashes will be updated with animation.
(I know our list lerps not implemented well, I will re-implement them in the near future)